pysd/py_backend/functions.py

```diff
@@ -267,7 +268,7 @@ def __init__(self, py_model_file, params=None, return_func=None):
         self.time = None
 
         # need a unique identifier for the imported module.
-        module_name = py_model_file + str(random.randint(0, 1000000))
+        module_name = os.path.splitext(py_model_file)[0] + str(random.randint(0, 1000000))
```

Comment:
I don't see anything wrong with this, but what does it do?

Reply:
The previous line generates the wrong name for python module and it generates some warning when model loaded.

Previous line return name of module like this: `simple-model-execution-process.py<RANDOM>` and it's wrong. The new line returns `simple-model-execution-prcess<RANDOM>`. This is correct for python.

But I'm not sure that it is not correct way to get name of the file without extension.

Reply:
Works for me. Thanks!